test/utils.go

```diff
@@ -525,9 +525,11 @@ func getAerospikeStorageConfig(
 	// TODO
 	ddInitMethod := asdbv1beta1.AerospikeVolumeInitMethodDD
 	blkDiscardInitMethod := asdbv1beta1.AerospikeVolumeInitMethodBlkdiscard
+	blkDiscardWipeMethod := asdbv1beta1.AerospikeVolumeWipeMethodBlkdiscard
```

Comment:
You can include the wipe method for dd and deleteFiles too.

Reply:
It's not added on purpose - to make sure that defaults are used correctly.